roles/edpm_nova/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+# Copyright 2023 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: configure nova
+  include_tasks: configure.yml
+
+- name: install nova
+  include_tasks: install.yml
+
+- name: Flush handlers
+  meta: flush_handlers
+
+- name: post-install nova
```

Comment:
The file is empty so nothing is executed in this step.

Reply:
yep that is intentional
i wanted to keep the pattern consistent